.changeset/spicy-papers-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'@spectrum-web-components/menu': patch
+---
+
+resolves issue with mouseover and focus styling
```

Comment:
I think the changesets can benefit from being formatted more like formal documentation than commit logs. This is a great opportunity to outline the updates being made just like in the pull request description and highlight specifically any changes consumers need to make adjustments for or be aware of during their validation of the update.

Reply:
Thanks, @castastrophe!